main.rb

```diff
@@ -0,0 +1,8 @@
+class Main
+  def run
+    puts 'Hello world!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.